config/application.yml.example

```diff
@@ -0,0 +1,4 @@
+# Remove the '.example' suffix from this file name and add AWS keys below
```

Comment:
Please put these instructions in the README.  They belong in the same list as the instructions about database.yml.